docs/UserGuide.md

```diff
@@ -158,6 +158,57 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+
+### Creating a new volunteer's profile : `vcreate` [coming soon]
+
+Volunteer Coordinators can create new volunteer profiles, and add the volunteer into the volunteer list.
+
+Format: `vcreate vn/VOLUNTEER_NAME hp/PHONE_NUMBER e/EMAIL [s/SKILLS]…`
+
+Parameters:
+ * vn/ - Volunteer name
+ * hp/ - Phone number of the volunteer
```

Comment:
I think we should stick to `p/` ? As per AB3

Reply:
suree, updated on my side